browser-test/src/applicant/applicant_application_index.test.ts

```diff
@@ -886,10 +886,25 @@ test.describe('applicant program index page with images', () => {
   test(
     'shows program with wide image in North Star and removes image from card when in My Applications',
     {tag: ['@northstar']},
-    async ({page, adminPrograms, adminProgramImage, applicantQuestions}) => {
+    async ({
+      page,
+      adminPrograms,
+      adminQuestions,
+      adminProgramImage,
+      applicantQuestions,
+    }) => {
       const programName = 'Wide Image Program'
       await loginAsAdmin(page)
+
+      await adminQuestions.addPhoneQuestion({
```

Comment:
**question (non-blocking):** I don't understand why we're adding this question to the test and how it's related to these bug fixes.  Could you explain?

Reply:
Oh, I see.  That makes sense now.  Rather than adding the question and answering it, what if the applicant directly clicked on "Submit application" from the review page?  Then, the card would still be put in the "My applications" section since the application would be complete.  I think that would detract less from the purpose of the test.  If that's a pain to do, then we can keep the new question.

Reply:
Oh, that works, and it makes the test simpler. Thanks for the suggestion :)